docs/source/api_reference/featurizers.rst

```diff
@@ -472,6 +472,30 @@ DummyFeaturizer
   :members:
   :inherited-members:
 
+Tokenizers
+----------
+
+GroverAtomVocabTokenizer
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: deepchem.feat.vocabulary_builders.grover_vocab.GroverAtomVocabTokenizer
+   :members:
+
+GroverBondVocabTokenizer
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass::  deepchem.feat.vocabulary_builders.grover_vocab.GroverBondVocabTokenizer
+   :members:
+
+Vocabulary Builders
+-------------------
```

Comment:
Vocabulary Builders are not featurizers so we should make this section a subsection of the tokenizer section since vocabulary builders are only used by tokenizers.

Can you add a description here about what a vocabulary builder is and why we want them in DeepChem?

Reply:
Added description in 7845f1d